tests/test_hw13_add_and_remove_termination_reason.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+
+from fixtures import AdminUserAuthentication
+from menus.top_nav import TopNavMenu
+from pages.add_termination_reason import AddTerminationReasonPage
+from pages.termination_reasons import TerminationReasonsPage
+
+
+class TerminationReasonTests(AdminUserAuthentication):
+
+    def test_termination_reason(self):
+        top_nav = TopNavMenu(self.browser)
+        termination_page = TerminationReasonsPage(self.browser)
+        add_termination_reason = AddTerminationReasonPage(self.browser)
+
+        top_nav.open_PIM_Configuration_Termination_reasons()
+        self.assertEqual(termination_page.HEADER, termination_page.get_page_header())
+        self.assertIn(termination_page.PAGE_URL, self.browser.current_url)
+        termination_page.click_add_btn()
+
+        self.assertEqual(add_termination_reason.HEADER, add_termination_reason.get_page_header())
+        self.assertIn(add_termination_reason.PAGE_URL, self.browser.current_url)
+        termination_reason = 'Missing / No - Show'
+        add_termination_reason.add_reason(termination_reason)
+
+        for reason in termination_page.get_list_of_reasons():
+            if termination_reason == reason.text:
+                return True
```

Comment:
This exits out of your test but does not fail the test if a reason is missing. You need an assertion instead.